src/MoBi.Core/Commands/AddBuildingBlocksToModuleCommand.cs

```diff
@@ -0,0 +1,125 @@
+﻿using MoBi.Assets;
+using MoBi.Core.Domain.Model;
+using MoBi.Core.Events;
+using MoBi.Core.Helper;
+using OSPSuite.Core.Commands.Core;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Utility.Extensions;
+
+namespace MoBi.Core.Commands
+{
+   public class AddBuildingBlocksToModuleCommand : MoBiReversibleCommand, ISilentCommand
+   {
+      protected Module _existingModule;
+      protected Module _moduleWithNewBuildingBlocks;
+      public string ExistingModuleId { get; private set; }
+      public string ModuleWithNewBuildingBlocksId { get; private set; }
+
+      public AddBuildingBlocksToModuleCommand(Module existingModule, Module moduleWithNewBuildingBlocks)
```

Comment:
Why is the use case a module to a module and not a buildingblock to a module
I am having trouble seeing how this is supposed to work